frontend/src/modules/tenant/store/state.js

```diff
@@ -4,6 +4,7 @@ export default {
     ids: [],
     loading: false
   },
+  saveLoading: false,
```

Comment:
Can we manage this loading state within respective components?
Or is this used/changed within multiple components at the same time?

Reply:
refactored